src/app/app.component.html

```diff
@@ -0,0 +1,6 @@
+<mat-toolbar class="container" color="primary" style="flex-grow: 5">Shop Easy
+  <div class="logout_container">
+    <button (click)="login()" color="accent" mat-raised-button>Logout</button>
```

Comment:
logout button should not be calling `login` method. Maybe call it `navigateToLogin`

Reply:
Done.